MinersLegacy/Excavator.ps1

```diff
@@ -43,7 +43,7 @@ $Name = Get-Item $MyInvocation.MyCommand.Path | Select-Object -ExpandProperty Ba
 $Devices = $Devices | Where-Object Type -EQ "GPU" | Where-Object Vendor -EQ "NVIDIA Corporation"
 
 $Devices | Select-Object Vendor, Model -Unique | ForEach-Object {
-    $Miner_Device = $Devices | Where-Object Vendor -EQ $_.Vendor | Where-Object Model -EQ $_.Model
+    $Miner_Device = $Devices | Where-Object Model -EQ $_.Model
```

Comment:
Where-Object Vendor -EQ $_.Vendor |
is redundant - is already in line 43

Reply:
Lets keep it in as to not transpose an error to other miners based off this.